include/zephyr/net/wifi_mgmt.h

```diff
@@ -229,7 +229,7 @@ struct wifi_twt_params {
 };
 
 /* Flow ID is only 3 bits */
-#define WIFI_MAX_TWT_FLOWS 8
+#define WIFI_MAX_TWT_FLOWS 7
```

Comment:
Changing this definition will affect the array size of the following structure.

`struct wifi_ps_config {`
`	struct wifi_twt_flow_info twt_flows[WIFI_MAX_TWT_FLOWS];`

Reply:
Yes, u[pdated the value and releated changes.